### ISSUE TITLE
Confusion of test result?(about test validation set split)

### ISSUE THREAD

**Issue by Liudzz** [weiaicunzai/pytorch-cifar100#60]
Firstly, thanks for sharing so many model on cifar which is very friendly to beginner ! I have learned CNN for 2 years and your project can always  inspire me !

I'd like you to ask you a question. In your implement **_train_** code ,  do you use test set to detemine your **_best model_**? If so, all the results got from best train model ? I have learned that datasets alway need to be divide  into ,"train/test/val",3 folds.
And what also confused me is the accs on Imagenet .Do they use val set to detemine best model and then test ? 
In my mind , you use test fold to test after train and that's your best acc . I also use this thought in my project and learning.   

I've searched on Internet and still have't find out . It would be great  if you can response me ! : )

Bty , In line 171 of your _**train.py**_ code , you use time as file name , if we train start from a certain epoch , we can't get a continuous train line .   In my project ,  I use _**runs**_  as a default name . And it seems that I can get a continuous line ?(Maybe I saved checkpoint and trained from a certrain epoch?)

I've noticed this project was been updated  5 month ago , and I'd like to thank you again !
